tools/devops/automation/templates/tests/build.yml

```diff
@@ -76,6 +76,9 @@ parameters:
   type: number
   default: 3
 
+- name: XcodeChannel
+  type: string
```

Comment:
Should we set a default value here?

Reply:
The default is going to be wrong for at least third of each year, so I think I prefer just erroring out if the caller forgets to pass this variable.